infrastructure/src/main/java/org/corfudb/infrastructure/management/failuredetector/ClusterGraph.java

```diff
@@ -212,6 +212,21 @@ private boolean isLocalNode(NodeConnectivity node) {
             return Optional.empty();
         }
 
+        Optional<NodeRank> fullyConnected = findFullyConnectedNode(
```

Comment:
Leave a comment with an example as for why this step is needed.

Reply:
done